licenses.yaml

```diff
@@ -1668,1285 +1604,1459 @@ libraries:
-license_name: CDDL 1.1
-version: 1.2.0
-copyright: Oracle and/or its affiliates.
-license_file_path: licenses/bin/javax.activation.CDDL11
-libraries:
-  - com.sun.activation: javax.activation
+module: web-console
+license_name: MIT License
+copyright: Alex Reardon
+version: 5.0.5
+license_file_path: licenses/bin/memoize-one.MIT
 
 ---
 
-name: Jersey
+name: "mini-create-react-context"
 license_category: binary
-module: java-core
-license_name: CDDL 1.1
-version: 1.19.3
-copyright: Oracle and/or its affiliates.
-license_file_path: licenses/bin/jersey.CDDL11
-libraries:
-  - com.sun.jersey: jersey-core
-  - com.sun.jersey: jersey-server
-  - com.sun.jersey: jersey-servlet
-  - com.sun.jersey: contribs
-  - com.sun.jersey.contribs: jersey-guice
+module: web-console
+license_name: MIT License
+copyright: StringEpsilon
+version: 0.3.2
+license_file_path: licenses/bin/mini-create-react-context.MIT
 
 ---
 
-name: Expression Language 3.0 API
+name: "ms"
 license_category: binary
-module: java-core
-license_name: CDDL 1.1
-version: 3.0.0
-copyright: Oracle and/or its affiliates.
-license_file_path: licenses/bin/javax.CDDL11
-libraries:
-  - javax.el: javax.el-api
+module: web-console
+license_name: MIT License
+version: 2.0.0
+license_file_path: licenses/bin/ms.MIT
 
 ---
 
-name: Java Servlet API
+name: "node-fetch"
 license_category: binary
-module: java-core
-license_name: CDDL 1.1
-version: 3.1.0
-copyright: Oracle and/or its affiliates.
-license_file_path: licenses/bin/javax.CDDL11
-libraries:
-  - javax.servlet: javax.servlet-api
+module: web-console
+license_name: MIT License
+copyright: David Frank
+version: 1.7.3
+license_file_path: licenses/bin/node-fetch.MIT
 
 ---
 
-name: JSR311 API
+name: "normalize.css"
 license_category: binary
-module: java-core
-license_name: CDDL 1.1
-version: 1.1.1
-copyright: Oracle and/or its affiliates.
-license_file_path: licenses/bin/jsr311-api.CDDL11
-libraries:
-  - javax.ws.rs: jsr311-api
+module: web-console
+license_name: MIT License
+version: 8.0.1
+license_file_path: licenses/bin/normalize.css.MIT
 
 ---
 
-name: Expression Language 3.0
+name: "numeral"
 license_category: binary
-module: java-core
-license_name: CDDL 1.1
-version: 3.0.0
-copyright: Oracle and/or its affiliates.
-license_file_path: licenses/bin/javax.CDDL11
-libraries:
-  - org.glassfish: javax.el
+module: web-console
+license_name: MIT License
+copyright: Adam Draper
+version: 2.0.6
+license_file_path: licenses/bin/numeral.MIT
 
 ---
 
-name: Jersey
+name: "object-assign"
 license_category: binary
-module: hadoop-client
-license_name: CDDL 1.1
-version: 1.9
-copyright: Oracle and/or its affiliates.
-license_file_path: licenses/bin/jersey.CDDL11
-libraries:
-  - com.sun.jersey: jersey-client
-  - com.sun.jersey: jersey-core
+module: web-console
+license_name: MIT License
+copyright: Sindre Sorhus
+version: 4.1.1
+license_file_path: licenses/bin/object-assign.MIT
 
 ---
 
-name: JavaBeans Activation Framework
+name: "path-to-regexp"
```

Comment:
Missing copyright: https://github.com/pillarjs/path-to-regexp/blob/master/LICENSE#L3